python/grass/imaging/images2gif.py

```diff
@@ -72,7 +72,7 @@
 
     pillow = True
     try:
-        PIL.__version__  # test if user has Pillow or PIL
+        PIL_version = PIL.__version__  # test if user has Pillow or PIL
```

Comment:
This returns a differen type (not bool True). I guess it does not make a differences though...

Reply:
It didn't test or return anything, it would have called if the statement was executed like in an interactive Python session. That was the warning. So here, at least I make it assigned to something (we can't use `_` to discard because of translation).
Even after this PR, it doesn't return anything (like before).

It was never supposed to be a Boolean.

Reply:
I have other smaller PRs (compared to last week) that you could take a look, I didn't request you specifically to not overload you. This one had multiple changes in temporal.